Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -20269,7 +20273,9 @@
 		};
 		5C400E6A29DB8AB500446F6F /* PBXTargetDependency */ = {
 			isa = PBXTargetDependency;
-			platformFilter = ios;
+			platformFilters = (
+				ios,
+			);
```

Comment:
Hm.. some more spurious Xcode project changes (though, these _should_ be innocuous)